cpp/src/slice2java/Gen.cpp

```diff
@@ -2735,53 +2728,33 @@ Slice::Gen::TypesVisitor::visitExceptionStart(const ExceptionPtr& p)
     out << sp;
     out << nl << "public " << name << "()";
     out << sb;
-    if (base)
-    {
-        out << nl << "super();";
-    }
```

Comment:
In Java, constructors will implicitly call `super();` if no other explicit call to `super` is made, no need to write it out.